packages/contracts/src/codegen/world/IDevSystem.sol

```diff
@@ -13,7 +13,7 @@ import { MaterialId } from "./../../libraries/Libraries.sol";
 interface IDevSystem {
   function graduate() external;
 
-  function fillTank(bytes32 _tankEntity, uint256 _amount, MaterialId _materialId) external;
+  function devFillTank(bytes32 _tankEntity, uint256 _amount, MaterialId _materialId) external;
```

Comment:
this function could be removed entirely to use a helper with `fillTank`, some admin pranking and minting instead